doc/contributing/PROCESS.md

```diff
@@ -174,6 +174,92 @@ To make a new OpenAssetIO release, follow this procedure.
   uploaded correctly.
 - You're done! Take a break and relax!
 
+## Breaking Integrations
+
+OpenAssetIO is pursuing an integration heavy validation strategy, it
+seeks to have a set of robust proof of concept projects, and will use
+these as both design validators and testing coverage providers.
+
+In order to tie this coverage into mainline OpenAssetIO development,
+OpenAssetIO continuous integration checks out several downstream
+projects each time a commit is pushed. These projects are built against
+the bleeding edge version of OpenAssetIO, and their test suites are run.
+This provides valuable end to end testing of each changeset, and is
+invaluable. (See
+[integrations.yml](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/.github/workflows/integrations.yml))
+
+Furthermore, as these downstream repos are projects in their own right,
+they also have their own CI suites to validate any changes made to them,
+using a released OpenAssetIO version, taken from PyPI.
+
+### Breaking Change Mitigation Process
+
+On occasion, it is necessary for OpenAssetIO to change its interfaces in
+such a way that cause the downstream projects to become incompatible
+against the bleeding edge OpenAssetIO. You will know when this happens
+as one of the tests in
+[integrations.yml](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/.github/workflows/integrations.yml)
+will begin to fail.
+
+The downstream projects are pinned to a specific version of OpenAssetIO
+on PyPI for their own test suites, so these will continue to function
+even through a breaking change on OpenAssetIO main. However, the
+OpenAssetIO integrations checks will being to fail, with no clear way to
+remedy the situation before an OpenAssetIO release, as any changes to
+the downstream project to support bleeding edge will likely cause
+incompatibility with the published OpenAssetIO release the project is
+consuming from PyPI.
+
+In order to remedy this, when making a breaking OpenAssetIO change that
+causes a failing integrations check, the following procedure may be
+implemented to avoid long term loss of coverage.
+
+- Consult on whether the change must necessarily be breaking, if
+reasonably possible, attempt to formulate another approach.
+
+- Evaluate the scope of breakage in the dependent project(s), and make
+an issue in those projects to fix compatibility with your change.
+
+- Comment out the failing integrations check in
+[integrations.yml](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/.github/workflows/integrations.yml).
+Leave a note explaining the context, with links to the relevant issues.
+
+- Review and merge your change to OpenAssetIO as normal, call out the
+removed integrations check in the PR.
+
+- As soon as possible, plan to work on the issues to return
+compatibility to the dependent repos.
```

Comment:
[consider] Be more specific?

```suggestion
- As soon as possible, plan to work on the issue(s) created in step 2 to 
return compatibility to the dependent repos.
```

Reply:
[44a9b4b](https://github.com/OpenAssetIO/OpenAssetIO/pull/772/commits/44a9b4b40357ae7eb8ee576cd93afb7b3d644e21)